### ISSUE TITLE
number of entities in fb237 is not the same with docs

### ISSUE THREAD

**Issue by Chen-Cai-OSU** [Accenture/AmpliGraph#99]
Hello,

I am trying to organize all triples into an actual KB graph in networkx. When I tried to do so, I double checked the number of entities and it seems there is some discrepancy between what is written in docs and what I got.

Here is the code to replicate the issue. fb237  is supposed to have 14,541 entities but I only got 14,505. Is there something I am missing?

    X = load_fb15k_237()
    entities, triples = [], []
    for X_ in X.values():  # X_ = X['train']
        for triple in X_:
            triples.append(list(triple))
            entities.append(triple[0])
            entities.append(triple[2])
    print(len(set(entities)))

**Comment by Chen-Cai-OSU** [comment]
btw, do you know how to get the entity name from its corresponding mid? I found [this](https://github.com/xiaoling/figer/issues/6), but not sure if this can cover all the entities.

**Comment by chanlevan** [comment]
Hello @Chen-Cai-OSU, 

Thanks for your point. Actually in fb15k-237, also in wn18rr and yago3-10, they contain unseen entities. What we showed in the dataset tables are all distinct entities including unseen ones.

If you go to `ampligraph/datasets/datasets.py` and modify `clean_unseen` flag to False in function `load_fb15k_237`, you will get the same result we indicated.

We will take this into account and add details in the dataset information table to avoid the confusion.
